taipy/core/taipy.py

```diff
@@ -873,6 +873,23 @@ def get_cycles() -> List[Cycle]:
     return _CycleManagerFactory._build_manager()._get_all()
 
 
+def can_create(config: Union[ScenarioConfig, DataNodeConfig]) -> Reasons:
```

Comment:
Is it possible to make the config optional?
If no config is passed, we just return True (an empty Reason). The false part will be implemented with the role management in Enterprise Edition.

So the API can be used for a specific config, or globally.

What do you think?

Reply:
I'm not sure to understand this. So the user can check if he can create anything at all when using enterprise?

Reply:
Yes. 
The purpose is deactivating the 'add scenario' button in the scenario selector.
At this stage, there is no configuration to pass since clicking on the button opens a dialog in which you can select your configuration and create a new scenario from it.
If I don't have the right to create a scenario (whatever the configuration), it is useless to let the user click on the 'add scenario' button.

To sum up, `can_create()` should return true if and only if I have the right to create a scenario or a data node.